### ISSUE TITLE
Opening or closing files takes from 20 up to 60 seconds

### ISSUE THREAD

**Issue by yodho1psnrls** [daic0r/dap-helper.nvim#2]
I suspect it is from the saved breakpoints loading

**Comment by daic0r** [comment]
Are you sure that's because of my plugin? I haven't been able to observe this.

**Comment by yodho1psnrls** [comment]
> Are you sure that's because of my plugin? I haven't been able to observe this.

Yes, i had noticed a massive lag when opening or closing files in neovim, also somethimes even when swapping buffers. So i though that it could be from my hard disk or neovim in general, but with other text editors i didnt had any problem with that, they opened instantly. Later i thought that it is because i open it from nvim-tree or telescope, but i also tried to open them just with the command :edit and the same problem still persisted, it was no different from nvim-tree or telescope. I have around 40 plugins installed on my neovim config and i decided to disable them all, i noticed that the files were opening just fine. Then i started enabling the plugins one by one, and at some point i had all my plugins enabled except the dap.lua file, which had nvim-dap and some other related plugins. I enabled it and the lag started again. So i went through the file, commented out the installation of dap-helper (i use Lazy) and i didnt noticed any lag at all. Now few days later with the same exact config, only except dap-helper, it works all fine, and i hadnt seen any issues with opening or closing files or tabbing through buffers. I very like your plugin, it is very convenient to be able to store some breakpoints that you always need at some places, but i was forced to disable it, because it is just so annoying to open a new file and to have to wait up to a minute for it to open, while the whole editor is frozen. My intuition is that the problem lies in how the saved breakpoints are loaded. Or it could actually conflict with some of the other plugins. And by the way, I am on Windows 10 (the problem may also be related to the OS)

**Comment by jmarcelomb** [comment]
It happens with me in ubuntu only, don't know why
